application/src/main/java/org/thingsboard/server/service/entitiy/alarm/DefaultTbAlarmService.java

```diff
@@ -196,7 +196,7 @@ public List<AlarmId> unassignDeletedUserAlarms(TenantId tenantId, User user, lon
     public Boolean delete(Alarm alarm, User user) {
         TenantId tenantId = alarm.getTenantId();
         logEntityActionService.logEntityAction(tenantId, alarm.getOriginator(), alarm, alarm.getCustomerId(),
-                ActionType.DELETED, user);
+                ActionType.ALARM_DELETE, user);
```

Comment:
We might break backward compatibility.

Reply:
It's a bug. It should be ALARM_DELETE.

Reply:
https://github.com/thingsboard/thingsboard/pull/10205